lib/easypost.rb

```diff
@@ -134,7 +134,7 @@ def self.parse_response(status:, body:, json:)
       )
     end
 
-    json ? JSON.parse(body) : body
+    json || status != 204 ? JSON.parse(body) : body
```

Comment:
nit: I hate it, but I'll allow it.

Reply:
Do you have an alternative that would be better? All for suggestions. Basically if the status is 204 (when we delete a user), there is no body at all so this blows up because it can't parse nothing, so instead we return body which is just empty.

Reply:
When we revisit the delete functions in all libs here shortly, we can clean this up since that will change this up a bit too.

Reply:
Detecting the `Content-Type` header and if the body is non-nil and non-empty is what we do internally.